### PR TITLE
Added files for Azure Load Testing

### PR DESCRIPTION
**Pull request by ninallam**

### CODE REVIEW

File: Test/config.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+testPlan: DemoPodcastTest.jmx
+description: ''
+engineInstances: 5
+configurationFiles:
+- jmeter-plugins-functions-2.2.jar
+- Shows.csv
+splitAllCSVs: True
+env:
+- name: webapp
+  value: dotnetpodcasts.azurewebsites.net
```
Comment (jamesmontemagno) [inline Test/config.yml:12]: Should this be our staging area? dotnetpodasts-staging.azurewebsites.net

File: .github/workflows/podcast-loadtest.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Podcast Load Test
+
+# Controls when the workflow will run
+on:
+  workflow_dispatch:
+
+env:
```
Comment (jamesmontemagno) [inline .github/workflows/podcast-loadtest.yml:9]: I am a bit worries about these hardcoded strings, we should pull from github secrets and document it.

**Comment by jamesmontemagno** [comment]
is there a license for the .jar file?

**Comment by ninallam** [comment]
@jamesmontemagno I've made the changes and added the demo. Could you please review?

**Review by jamesmontemagno — APPROVED** [review]

**Comment by jamesmontemagno** [comment]
Will merge in @ninallam are you able to have them update links on the videos and stuff to point here to the main repo.

**Comment by ninallam** [comment]
> Will merge in @ninallam are you able to have them update links on the videos and stuff to point here to the main repo.

Thanks @jamesmontemagno. I have updated the links.

**Comment by donatocayurin** [comment]
❤️🙏

**Pull request merged**